QUICKSTART.md

```diff
@@ -181,13 +188,17 @@ HttpTextFormat.Setter<HttpURLConnection> setter =
 
 URL url = new URL("http://127.0.0.1:8080/resource");
 Span outGoing = tracer.spanBuilder("/resource").setSpanKind(Span.Kind.CLIENT).startSpan();
-// Semantic Convention
-outGoing.setAttribute("http.method", "GET");
-outGoing.setAttribute("http.url", url.toString());
-HttpURLConnection transportLayer = (HttpURLConnection) url.openConnection();
-// Inject the request with the context
-tracer.getHttpTextFormat().inject(outGoing.getContext(), transportLayer, setter);
-// Make outgoing call
+try (Scope scope = tracer.withSpan(outGoing)) {
+  // Semantic Convention
+  outGoing.setAttribute("http.method", "GET");
+  outGoing.setAttribute("http.url", url.toString());
```

Comment:
This makes it look like if setting attributes was only allowed within the scope. You might want to move that out to make it clear this is not the case.

Reply:
I'd rather add a note such as `setting attributes does not require the Span to be associated to the current Context`, as moving that out of the scope would also give users a bad idea (that you set the attributes first *and then* you set the `Span` active). Would that work for you?

Reply:
Sounds reasonable!

Reply:
Done.